about/diversity.md

```diff
@@ -22,12 +22,12 @@ This is an ongoing process. We are talking to our program chairs, program commit
 
 Here are some ways you can help us build a more diverse conference experience:
 
-* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pycon-staff.
+* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pytn-staff.
```

Comment:
We should probably change these references to "pytn-staff" to "[organizers@pytennessee.org](mailto:organizers@pytennesee.org)". We don't have a `pytn-staff@` email address...

Reply:
Sounds good. I can work on that tomorrow.